src/protocols/ssh/settings.h

```diff
@@ -253,6 +253,10 @@ typedef struct guac_ssh_settings {
      * environment variable.
      */
     char* locale;
+    /** 
```

Comment:
Missing a blank line separating the block comment and the code above.

Reply:
Added.